eng/pipelines/extra-platforms/runtime-extra-platforms-androidemulator.yml

```diff
@@ -118,3 +118,35 @@ jobs:
           parameters:
             creator: dotnet-bot
             testRunNamePrefixSuffix: Mono_$(_BuildConfig)
+
+#
+# Android emulators
+# Build the whole product using CoreCLR and run libraries tests
```

Comment:
Are these library tests? I thought that functional tests are usually under the runtime test suit.

Reply:
Functional tests are under libraries. There might be smoke tests under the runtime tests.

Reply:
Ah, I thought it was the reverse. The https://github.com/dotnet/runtime/blob/main/src/tests/FunctionalTests/Android/Device_Emulator/JIT/Android.Device_Emulator.JIT.Test.csproj is placed under `src/tests` so I probably got mistaken that it is part of runtime tests. 

In case of smoke tests, I think these run functional tests and `System.Runtime.Tests` which comes from libraries.